services/scheduler/src/main/java/org/mule/service/scheduler/internal/AbstractRunnableFutureDecorator.java

```diff
@@ -66,6 +69,35 @@ protected long beforeRun() {
     return startTime;
   }
 
+  protected void doRun(RunnableFuture<V> task, ClassLoader classLoader) {
+    long startTime = beforeRun();
+
+    final Thread currentThread = currentThread();
+    final ClassLoader currentClassLoader = currentThread.getContextClassLoader();
+    currentThread.setContextClassLoader(classLoader);
+
+    try {
+      task.run();
+      task.get();
+    } catch (ExecutionException e) {
+      logger.error("Uncaught throwable in task " + toString(), e);
+    } catch (CancellationException e) {
+      logger.trace("Task " + toString() + " cancelled");
```

Comment:
No. That was just added no not clutter the log/console with cancellation stack traces. Reactor cancells jobs all the time and it fills the log